app/class-option/page.js

```diff
@@ -1,3 +1,66 @@
+"use client";
+import ClassCard from "../components/ClassCard";
+import { useRouter } from "next/navigation";
+
 export default function ClassSelection() {
-  return <p>This is the class option page.</p>;
+  const router = useRouter();
+
+  const classes = [
```

Comment:
Can we move the classes and characters constants into a data file and place this in the `lib` directory to clean this up a bit?

Reply:
got it